app/policies/person_policy.rb

```diff
@@ -0,0 +1,24 @@
+class PersonPolicy < ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def edit?
```

Comment:
we get this from ApplicationPolicy? not sure what standard is for adding to child policies

Reply:
I think originally I had changed it to something else and was just changing it back here. I realize it's redundant, but it also explicitly declares the expected behavior (were `ApplicationPolicy` to be changed later, that would create problems here). My personal preference for stuff like this is to have the default behaviors as a safety net but to explicitly define things that are definitely used. 

I'm happy to defer to yours / the team's preferences on this though!